feature-group-definitions/container-style-queries.dist.yml

```diff
@@ -0,0 +1,15 @@
+# Generated from: container-style-queries.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Container style queries
+spec: https://drafts.csswg.org/css-contain-3/#style-container
+caniuse: css-container-queries-style
+usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/4550
+status:
+  baseline: false
+  support:
+    chrome: "111"
```

Comment:
Yes: https://developer.mozilla.org/en-US/docs/Web/CSS/@container#browser_compatibility

Reply:
https://caniuse.com/css-container-queries-style agrees too, only difference is that it calls out that it's limited to custom properties. I suspect it will remain like that ~indefinitely and that'll just be what the feature is.

Reply:
I put a suggestion relating to this here: https://github.com/web-platform-dx/web-features/pull/751/files#r1557476030